### ISSUE TITLE
First iteration of Resource Health BB as a service offer

### ISSUE THREAD

**Issue by dovydas-an** [EOEPCA/resource-health#13]
Publish first iteration of Resource Health BB specifications as a service offer, highlighting benefits and business value of the BB.

**Comment by rconway** [comment]
@dovydas-an Is this reflected in the BB documentation?

**Comment by dovydas-an** [comment]
@rconway yes, this has been included here: https://eoepca.readthedocs.io/projects/resource-health/en/latest/service-offer/
